WordPress/Classes/Utility/Media/CachedAsyncImage.swift

```diff
@@ -30,8 +31,8 @@ struct CachedAsyncImage<Content>: View where Content: View {
 
     /// Allows content customization and providing a placeholder that will be shown
     /// until the image download is finalized.
-    init<I, P>(url: URL?, @ViewBuilder content: @escaping (Image) -> I, @ViewBuilder placeholder: @escaping () -> P) where Content == _ConditionalContent<I, P>, I: View, P: View {
-        self.init(url: url) { phase in
+    init<I, P>(url: URL?, host: MediaHost? = nil, @ViewBuilder content: @escaping (Image) -> I, @ViewBuilder placeholder: @escaping () -> P) where Content == _ConditionalContent<I, P>, I: View, P: View {
```

Comment:
Not related to this PR, but it's pretty strange to give an optional URL to a type whose job is loading an image from an URL.

But, what do you think making `host` non-optional. Because `MediaHost` support public and private sites. Declare host as optional, would make it possible to write something like `CachedAsyncImage(postFromAPrivateSite.featuredImage) { ... }`, which would fail to load.

Reply:
> it's pretty strange to give an optional URL to a type whose job is loading an image from an URL.

It's modeled after [AsyncImage](https://developer.apple.com/documentation/swiftui/asyncimage/init(url:scale:content:placeholder:)). If the URL is nil, it shows a placeholder, so it's pretty convenient.

> But, what do you think making host non-optional.

You need to be able to use it for loading images outside of WordPress blogs and `MediaHost`.